rules/windows/process_creation/proc_creation_win_sc_sdset_allow_service_changes.yml

```diff
@@ -0,0 +1,38 @@
+title: Allow Service Access Using Security Descriptor Tampering Via Sc.EXE
+id: 6c8fbee5-dee8-49bc-851d-c3142d02aa47
+related:
+    - id: a537cfc3-4297-4789-92b5-345bfd845ad0 # Generic SD tampering
+      type: similar
+status: experimental
+description: Detects suspicious DACL modifications to allow access to a service from suspicious trustee. This can be used to override access restrictions set by previous ACLs
```

Comment:
```suggestion
description: Detects suspicious DACL modifications to allow access to a service from a suspicious trustee. This can be used to override access restrictions set by previous ACLs.
```